theme/styles/settings/_typography.scss

```diff
@@ -81,6 +81,17 @@
   }
 }
 
+.serif-sm-md,
+%serif-sm-md {
+  font-size: 2rem;
+  line-height: 2.1rem;
+
+  @include media('lg-up') {
+    font-size: 3.125rem;
+    line-height: 3.125rem;
+  }
+}
+
```

Comment:
I've been using the type names from the design file so it's easier to find, but sometimes in the designs the types classes are not used and new ones are created. So here I was trying to make an inbetween serif-sm and serif-md!

Reply:
haha yeah, it happens.